src/project/virtual_packages.rs

```diff
@@ -47,7 +49,11 @@ pub(crate) fn get_minimal_virtual_packages(
     }
 
     if platform.is_windows() {
-        virtual_packages.push(VirtualPackage::Win);
+        // todo: add windows to system requirements
```

Comment:
is it okay if i leave this open for another eventual PR?
this pr is already quite large